### PR TITLE
updated sku-config.yaml

### PR DESCRIPTION
**Pull request by makslion**
# Description
https://issues.redhat.com/browse/MGDAPI-1390

### Verification steps 
- Compare values in the .yaml with corresponding from a ticket linked above 

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist
- [ ] This change requires a documentation update 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added a test case that will be used to verify my changes 
- [ ] Verified independently on a cluster by reviewer

### CODE REVIEW

File: config/configmap/sku-config.yaml [hunk 1]
```diff
@@ -4,89 +4,243 @@ metadata:
...
+                            "memory": "500Mi"
+                        }
+                    }
+                },
+                "backend_worker": {
+                    "replicas": 3,
+                    "resources": {
+                        "requests": {
+                            "cpu": 0.15,
+                            "memory": "100Mi
```
Comment (laurafitzgerald) [inline config/configmap/sku-config.yaml:111]:
```suggestion
                            "memory": "100Mi"
```

**Comment by laurafitzgerald** [comment]
@makslion this looks fine. we can verify that the values are propagated as expected for each configuration as a verification step.

**Comment by makslion** [comment]
Discussed with @laurafitzgerald -> propagation of values will be verified as part of each product PR. This one only needs to have correct values in the .yaml

**Comment by KevFan** [comment]
Overall looks good :+1: . Just have 2 questions:

1. Grafana block seems to still in the 5 million and 1 million config, but removed from the rest, wondering are they still needed or kept for a reason ? (ticket does not have values for grafana) :thinking: 
2. Is there a list of expected `requests_per_unit` value for each config to compare or is okay to just multiple that value by `60 * 24` ? They were aproximately equal to the total sku value but just wondering is there a formal list to compare against :thinking:

**Comment by makslion** [comment]
1. Updated the config. Well spotted. It was an artefact and there is no need to keep it there now. 
2. `requests_per_unit` = SKU_Quota / 24 / 60 -> this is how they were calculated, yes. They are approximately equal to the SKU quota, but showing requests per minute (while SKU quota is per day and this is the only "formal list of requirements")

**Comment by KevFan** [comment]
Thanks @makslion 
/lgtm

**Pull request merged**